lib/assigner/include/vm_host.hpp

```diff
@@ -285,7 +285,8 @@ class VMHost : public evmc::Host
         auto hash = nil::blueprint::zkevm_word<BlueprintFieldType>(ethash::keccak256(msg.input_data, msg.input_size));
         auto sender = nil::blueprint::zkevm_word<BlueprintFieldType>(msg.sender);
         auto sum = nil::blueprint::zkevm_word<BlueprintFieldType>(0xff) + seed + hash + sender;
-        auto rehash = ethash::keccak256(sum.raw_data(), nil::blueprint::zkevm_word<BlueprintFieldType>::size);
+        auto rehash = ethash::keccak256(reinterpret_cast<const uint8_t*>(&sum),
```

Comment:
here casting to `uint8_t*` pointer to `zkevm_word structure`, but should be pointer to `zkevm_word::value`.
Sure pointer to the structure always pointer to the first non static field, but if we'll add another fields into `zkevm_word` - may get issues, which hard to debug
May be use `zkevm_word::get() `here

Reply:
Done